src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -105,6 +106,61 @@
             });
         }
 
+        #createTableSearchFields() {
+            document.querySelectorAll('div.table-with-search-field').forEach((table, i) => {
+                const searchField = document.createElement('input');
+                searchField.type = 'search';
+                searchField.placeholder = 'search...';
+                searchField.id = `table-search-field-${i}`;
+                searchField.classList.add(`table-search-field-input`);
+                searchField.autocapitalize = 'off';
+                searchField.autocomplete = 'off';
+                searchField.autocorrect = 'off';
+                table.insertBefore(searchField, table.firstChild);
+
+                const labelField = document.createElement('label');
+                labelField.htmlFor = `table-search-field-${i}`;
+                labelField.classList.add(`table-search-field-label`);
+                labelField.innerHTML = 'Search inside the contents of the table';
```

Comment:
use `.textContent` instead which avoids the need to escape HTML (even though this particular string does not require any escaping, this might change in the future)